composer.json

```diff
@@ -24,7 +24,8 @@
         "php": ">=7.1",
         "pimple/pimple": "^3.0",
         "aura/sqlquery": "^2.6",
-        "doctrine/cache": "^1.6"
+        "doctrine/cache": "^1.6",
+        "brick/geo": "^0.5.0"
```

Comment:
I think it should be moved to suggest

Reply:
Not sure how to handle unit test correctly with this moved to suggest
Is my implementation good ?

Reply:
You can add it in require-dev too, so it will be installed when working in ting, but not automatically for projects requiring ting (even in dev env)

Reply:
Should be good, thanks